examples/proto_dice/tests/proto_dice_tests.cpp

```diff
@@ -342,6 +390,11 @@ BOOST_FIXTURE_TEST_CASE(signidice_2_bad_state_test, proto_dice_tester) try {
     ), wasm_assert_msg("state should be 'req_signidice_part_2'"));
 
 } FC_LOG_AND_RETHROW()
+
+BOOST_FIXTURE_TEST_CASE(proto_dice_rtp_test, proto_dice_tester, *boost::unit_test::disabled()) try {
+    BOOST_TEST(get_rtp(1000000, 50) == 0.67, boost::test_tools::tolerance(0.02));
```

Comment:
I'm too lazy to verify that. For the real dice RTP should be `1 - house_edge` I suppose.

Reply:
But it's not real dice.